selfdrive/controls/controlsd.py

```diff
@@ -643,7 +642,6 @@ def publish_logs(self, CS, start_time, actuators, lac_log):
     controlsState.longitudinalPlanMonoTime = self.sm.logMonoTime['longitudinalPlan']
     controlsState.lateralPlanMonoTime = self.sm.logMonoTime['lateralPlan']
     controlsState.enabled = self.enabled
-    controlsState.active = self.active
```

Comment:
no need to remove this

Reply:
the only diff here should be longActive and latActive

Reply:
Should both need to be active for this? If so, when disengage on gas is false, controlsState can be partially active.